src/main/java/com/eleks/academy/whoami/core/impl/TurnImpl.java

```diff
@@ -52,6 +53,29 @@ public List<List<PlayerAction>> getTurns() {
 		return this.turns; //TODO: history
 	}
 
+	@Override
+	public boolean isQuestionPresent() {
+		return this.getCurrentTurn().get(0).getValue() != null;
+	}
+
+	@Override
+	public boolean isAnswerer(String player) {
+		int index = this.getCurrentTurn()
```

Comment:
I wouldn't really on array element index, we have the source of truth it is the action of player, if player's action is ANSWER then ok